app/services/symphony_client.rb

```diff
@@ -163,15 +162,20 @@ def renew_item_request(resource, item_key, headers: {})
                             })
     end
 
-    def error_message(response)
-      JSON.parse(response.body).dig('messageList')[0].dig('message')
+    def error_code(response)
+      return if response.status.ok?
+
+      JSON.parse(response.body).dig('messageList')[0].dig('code')
```

Comment:
Probably could make a little mini model out of `SymphonyResponse`. Just thinking outloud. Not a request for change at all.

Reply:
Might even be added to PlaceHoldResults::Builder 🤔 (which is added in the next PR)